for_developers/run_the_tests/index.md

```diff
@@ -7,26 +7,63 @@ sidebar:
   nav: "for_developers"
 ---
 
-Running all ROOT tests on at least a platform before sending a PR or before pushing anything to the ROOT repository is *mandatory*.
+Running all ROOT tests on at least a platform before sending a PR or before pushing anything to the ROOT repository is **mandatory**.
 The ROOT continuous integration infrastructure allows to automatically check the PR also on a battery of platform, for example including macos and linux boxes.
 In order to be able to run all tests, the simple steps below need to be followed
 
-## Prepare your development area and build ROOT
+## Prepare Your Development Area for Testing and Building ROOT
 
+#### Option 1: Build roottest as part of a ROOT build
```

Comment:
I would prefer that we don't offer options: we should keep at as simple as possible. And we should use as few words as possible: people don't come here to be entertained :-) Can you adjust this sightly, picking the option you prefer?

Reply:
Sure, will keep option 1: build roottest along with ROOT build